src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.scss

```diff
@@ -169,6 +169,17 @@ label {
   }
 }
 
+.signature-box {
+  display: inline-block;
+  margin-bottom: 15px;

```

Comment:
Can we use **rem** rather than **px**

Reply:
Why should we use rem rather than px? There are lots of other elements in the same file styled by px.

Reply:
**rem** gives you more responsive design style and it's scalable. Whenever we encounter **px**, slowly all the new and old css files are moved into **rem**.

Reply:
Good to know. Thanks. Done.